index.bs

```diff
@@ -2426,6 +2426,23 @@ Council Participation, Dismissal, and Renunciation</h5>
 	The [=W3C Council=] <em class=rfc2110>may</em> still solicit and hear their testimony,
 	as they can of anyone else in the W3C community.
 
+<h5 id=council-short-circuit>
+Unanimous Short Circuit</h5>
+
+
+	The full Council process <em class=rfc2119>may</em> be short-circuited if
+	the Team recommends a resolution
+	and every potential member of the Council who is not renouncing their seat
+	votes affirmatively (no abstentions) to adopt this resolution.
+
+	This step <em class=rfc2119>may</em> be run concurrently with [[#council-participation]]
+	and prior to choosing a [=W3C Council Chair|Chair=].
+
+	Note: This is intended for exceptional cases
+	that seem to have an obvious resolution
+	and which are too trivial or repetitive
```

Comment:
is this 'and' or 'or'?

Reply:
Which `and` or `or` are you asking about?
```
	that seem to have an obvious resolution
	and/or which are too trivial or repetitive
```
— or —
```
	that seem to have an obvious resolution
	and which are too trivial and/or repetitive
```
— or —
```
	that seem to have an obvious resolution
	and/or which are too trivial and/or repetitive
```
— or —
```
	that seem to have an obvious resolution, are
	too trivial, and/or are too repetitive
```
— or, since "obvious resolution" seems redundant with "trivial", my favorite —
```
	that are
	too trivial and/or repetitive
```

Reply:
The 'and' at the beginning, as you surmised; the condition seems to be `(has obvious resolution) AND (is trivial or repetitive)`, whereas I suspect there may be matters with obvious resolutions that are not trivial or repetitive (e.g. something may be momentous and complex, but out of our scope, and hence has the obvious resolution that we shouldn't touch something out of our scope). Not major, but 'or' or 'and/or' would seem safer to me.

Reply:
On the one hand, I agree that "trivial or repetitive" is an inexhaustive list of why we might want to take this path.

On the other hand, I think "obvious resolution" by itself isn't a good enough reason to take the fast path, because there might be a resolution that's obvious from the Team report, but if the Council were called and asked to deliberate, we might find some merit in the case that was not clear.

I don't want the Council to start forming a habit of dismissing FOs because they're a bit too checked out to dedicate time to digging into the issue and would rather ratify whatever the Team came up with.

I'm not sure yet how to word this better... maybe

```
Note: This is intended for exceptional cases
that don't seem to warrant a full Council response
because they are e.g. too trivial or repetitive or absurd.
```

?

Reply:
```
Note: This is intended for exceptional cases
that don't seem to warrant a full Council response
because they are, for instance, too trivial, repetitive, absurd, etc.
```
?

Reply:
took @TallTed 's suggested variant on @fantasai's proposal.